<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TLWindow</class>
 <widget class="QMainWindow" name="TLWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>391</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>350</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actTableInsert"/>
   <addaction name="separator"/>
   <addaction name="actTableDelete"/>
   <addaction name="separator"/>
   <addaction name="actToTree"/>
   <addaction name="separator"/>
   <addaction name="actTreeInsert"/>
   <addaction name="separator"/>
   <addaction name="actTreeDelete"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actBack"/>
   <addaction name="separator"/>
  </widget>
  <action name="actBack">
   <property name="text">
    <string>back</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actTableInsert">
   <property name="text">
    <string>线性表插入</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actTableDelete">
   <property name="text">
    <string>线性表删除</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actTreeInsert">
   <property name="text">
    <string>树插入</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actTreeDelete">
   <property name="text">
    <string>树删除</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actToTree">
   <property name="text">
    <string>由表生成树</string>
   </property>
   <property name="toolTip">
    <string>由表生成树</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
